releases.yml

```diff
@@ -1,4 +1,11 @@
 releases:
+  4.15.0:
+    assembly:
+      basis:
+        assembly: rc.8
+      group:
+        upgrades!: 4.14.14,4.15.0-ec.0,4.15.0-ec.1,4.15.0-ec.2,4.15.0-ec.3,4.15.0-rc.0,4.15.0-rc.1,4.15.0-rc.2,4.15.0-rc.3,4.15.0-rc.4,4.15.0-rc.5,4.15.0-rc.7,4.15.0-rc.8
```

Comment:
is the `!` in `upgrades!` significant?  I don't see other examples of that punctuation in this file.

Reply:
yes, it's basically a directive to force override
similar for 4.14.0 ga https://github.com/openshift-eng/ocp-build-data/blob/d167b52fa20d6fb3b35d5bcad3dc9fa190df7c78/releases.yml#L760